soroban-sdk/src/token.rs

```diff
@@ -0,0 +1,94 @@
+//! Token contains types for calling and accessing token contracts, including
+//! the Stellar Asset Contract.
+//!
+//! See [`Interface`] for the interface of token contracts such as the Stellar
+//! Asset Contract.
+//!
+//! See [`Client`] for calling token contracts such as the Stellar Asset
+//! Contract.
+use crate as soroban_sdk;
```

Comment:
@sisuresh This is a little trick so that we can use the macros inside the sdk. We just alias `crate` to `soroban_sdk` so that any generated code in this file that references the `soroban_sdk` will reference the types in this crate.